moonstone/utils/df_reindex.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from typing import Union
+
+from moonstone.utils.taxonomy import TaxonomyCountsBase
+
+
+class GenesToTaxonomy(TaxonomyCountsBase):
+
+    def __init__(self, dataframe: Union[pd.Series, pd.DataFrame],
+                 taxonomy_file: str = None, taxa_column: str = 'full_tax'):
+        self.df = dataframe
+        self.taxonomy_file = taxonomy_file
+        self.taxa_column = taxa_column   # noqa
```

Comment:
I think the `# noqa` is not necessary here?

Reply:
True, it was at some point. I forgot to remove it :)